awx/api/views/__init__.py

```diff
@@ -452,7 +452,8 @@ def post(self, request, *args, **kwargs):
                     else:
                         time.sleep(1.0)
                 else:
-                    obj.mark_offline(errors=_('Health check initiated by user determined this instance to be unresponsive'))
+                    if obj.capacity > 0:
+                        obj.mark_offline(errors=_('Health check initiated by user determined this instance to be unresponsive'))
```

Comment:
We also don't want to change `errors` if the instance already started out offline. This is a tricky case, but should be rarely relevant.